From: Maximiliano Curia <maxy@gnuservers.com.ar>
Date: Thu, 13 Jul 2017 17:39:47 +0200
Subject: Build docs if doxygen is available

---
 doc/doc.pri | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/doc/doc.pri b/doc/doc.pri
index 43229ff..f617019 100644
--- a/doc/doc.pri
+++ b/doc/doc.pri
@@ -24,7 +24,8 @@ DOC_INPUT += doc/src
 
 # target for generating documentation
 doctarget.target     = docs
-doctarget.commands   = OUTPUT_DIRECTORY=doc \
+doctarget.commands   = ! which doxygen || \
+                       OUTPUT_DIRECTORY=doc \
                        PROJECT_NAME=\"$${PROJECT_NAME}\" \
                        PROJECT_NUMBER=\"$${PROJECT_VERSION}\" \
                        STRIP_FROM_PATH=\"$${_PRO_FILE_PWD_}\" \
